src/Schema/AST/ASTBuilder.php

```diff
@@ -81,17 +81,12 @@ public function documentAST(): DocumentAST
         if (! isset($this->documentAST)) {
             $cacheConfig = $this->configRepository->get('lighthouse.cache');
             if ($cacheConfig['enable']) {
-                /** @var \Illuminate\Contracts\Cache\Factory $cacheFactory */
-                $cacheFactory = app(CacheFactory::class);
-                $cache = $cacheFactory->store($cacheConfig['store'] ?? null);
-
-                $this->documentAST = $cache->remember(
-                    $cacheConfig['key'],
-                    $cacheConfig['ttl'],
-                    function (): DocumentAST {
-                        return $this->build();
-                    }
-                );
+                $path = $cacheConfig['path'];
```

Comment:
The way Laravel merges config files we _need_ this fallback, because if the user has previously published their config they will not have the new `path` entry.

```suggestion
                $path = $cacheConfig['path'] ?? base_path('bootstrap/cache/lighthouse-schema.php');
```

Reply:
good thinking!

Reply:
I thought it was recursive, but it's just a simple array_merge on the 'lighthouse' config key level. 👍